airflow/models/abstractoperator.py

```diff
@@ -58,6 +58,8 @@
 DEFAULT_OWNER: str = conf.get_mandatory_value("operators", "default_owner")
 DEFAULT_POOL_SLOTS: int = 1
 DEFAULT_PRIORITY_WEIGHT: int = 1
+# TODO: Is None the right thing here? We want NULL essentially
```

Comment:
Looking for feedback on this, thanks!

Reply:
Why do we want this to be NULL?

Reply:
Good question, it's covered in more detail in the AIP (I wish I could hyper link to one specific section or comment...) but essentially NULL, means that no specific executor was requested for the task. In those cases we fall back to the historical/default behaviour of Airflow, which is to use whatever the currently configured environment level executor is, which may change at any time (if the user decides to switch it).